src/tasks/run.js

```diff
@@ -290,7 +292,7 @@ const configureApp = (app) => {
   }
 };
 
-if (isSandboxLinked()) {
+if (isSandboxLinked() && !process.env.SKIP_DEV_SERVER) {
```

Comment:
Did you mean to leave this check in, and is it more for internal testing or something to be documented publicly?

Reply:
@brenthosie It is more for internal testing. Sometimes I don't need the dev server to start and in order to make everything faster I was always commenting the line 296. I don't think this needs to be documented. The sandbox is linked only when we develop new features.